src/utils/dom.test.ts

```diff
@@ -16,4 +17,24 @@ describe('DOM utilities', () => {
     // Assuming the tests run only with Karma, which doesn't run tests inside an iframe
     expect(isAnyParentCrossOrigin()).toBeFalse()
   })
+
+  describe('Document loading - consistency test', () => {
```

Comment:
Jasmine concatenates all the ancestor description into 1 sentence.

Please also add a describe level telling that it tests withIframe.

For example:

```js
describe('DOM utilities', () => {
  describe('withIframe', () => {
    describe('with null document.body', () => {
      it("doesn't fail", () => {
```
The resulting test sentence will be "DOM utilities withIframe with null document.body doesn't fail".

Reply:
Sorry, the current nesting is excessive. Could you please do like this:

```js
  describe('withIframe', () => {
    it("doesn't fail when document loading is delayed", async () => {
      try {
        jasmine.clock().install()

        // The actual test

      } finally {
        jasmine.clock().uninstall()
      }
    })
  })
```